x-pack/legacy/plugins/siem/cypress/README.md

```diff
@@ -199,19 +199,19 @@ The current sets of data can be found in: `x-pack/test/siem_cypress/es_archives`
 
 ### How to generate new test data
 
-We are using es_archiver in order to generate the data that our Cypress tests needs.
+We are using archiver in order to generate the data that our Cypress tests needs.
```

Comment:
Ditto on the above; this is somewhat confusing and makes our dependencies (`scripts/es_archiver`) less discoverable.